test/cpp/microbenchmarks/helpers.cc

```diff
@@ -37,11 +39,16 @@ void TrackCounters::AddToLabel(std::ostream &out, benchmark::State &state) {
     out << " " << grpc_stats_counter_name[i]
         << "/iter:" << ((double)stats.counters[i] / (double)state.iterations());
```

Comment:
Should we use counters here?

Reply:
It looks like custom counters get printed to the json file as integers, so for our per-iter numbers we can't use them.